tests_integration/__tests__/__snapshots__/piped-output.js.snap

```diff
@@ -0,0 +1,15 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`no file diffs with --list-different + formated file 1`] = `Array []`;
+
+exports[`no file diffs with --list-different + formated file 2`] = `Array []`;
+
+exports[`output with --list-different + unformated differs when piped 1`] = `
+"unformated.js[2K[1Gunformated.js
```

Comment:
Can you add `--no-color` to these tests? We've had CI failures because of this before. (Also, the tests become more readable.)